app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/test/chapter5/StubAllIntentsTest.kt

```diff
@@ -52,8 +57,10 @@ class StubAllIntentsTest {
                 .respondWith(Instrumentation.ActivityResult(Activity.RESULT_OK, null))
     }
 
+    //Exercise 16.2 - Verify the share intent functionality based on the intent type and action
     @Test
     fun stubsShareIntent() {
+        val allExpectedIntent = (allOf(hasAction(Intent.ACTION_SEND),(hasType("text/plain"))))
```

Comment:
This name doesn't suggest what type of Intent we are trying to match - as I understand from exercise it is a share text intent correct?

Reply:
Yes, I should change this name.